docs/pages/radiobutton.js

```diff
@@ -13,335 +11,18 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
-          <MainSection.Card
-            cardSize="md"
-            type="do"
-            title="When to use"
-            description={`
-          - Situations where users can only choose one out of multiple, related options.
-        `}
-          />
-          <MainSection.Card
-            cardSize="md"
-            type="don't"
-            title="When not to use"
-            description={`
-          - Any time users can choose more than one option. Use [Checkboxes](/checkbox) instead.
-          - As a solitary option. RadioButtons should always appear in groups of 2 or more. Consider a [Checkbox](/checkbox) or [Switch](/switch) instead.
-        `}
-          />
-        </MainSection.Subsection>
-      </MainSection>
-      <Example
-        description="
-    Here is an example of an accessible group of radio buttons. All radio buttons should be wrapped in a [Fieldset](/fieldset).
-  "
-        name="RadioButton Group"
-        defaultCode={`
-function RadioButtonExample() {
-  const [favorite, setFavorite] = React.useState(undefined);
-  const [favoriteFood, setFavoriteFood] = React.useState(undefined);
-
-  return (
-    <Flex gap={4}>
-      <Fieldset legend="What is your favorite pet?">
-        <Flex direction="column" gap={2}>
-          <RadioButton
-            checked={favorite === 'dogs'}
-            id="favoriteDog"
-            label="Dogs"
-            name="favorite"
-            onChange={() => setFavorite( 'dogs' )}
-            value="dogs"
-          />
-          <RadioButton
-            checked={favorite === 'cats'}
-            id="favoriteCat"
-            label="Cats"
-            name="favorite"
-            onChange={() => setFavorite( 'cats' )}
-            value="cats"
-          />
-          <RadioButton
-            checked={favorite === 'plants'}
-            id="favoritePlants"
-            label="Plants"
-            name="favorite"
-            onChange={() => setFavorite( 'plants' )}
-            value="plants"
-          />
-        </Flex>
-      </Fieldset>
-      <Fieldset legend="What is your favorite food?">
-        <Flex gap={2}>
-          <RadioButton
-            checked={favoriteFood === 'pizza'}
-            id="favoritePizza"
-            label="Pizza"
-            name="favoriteFood"
-            onChange={() => setFavoriteFood( 'pizza' )}
-            value="pizza"
-          />
-          <RadioButton
-            checked={favoriteFood === 'curry'}
-            id="favoriteCurry"
-            label="Curry"
-            name="favoriteFood"
-            onChange={() => setFavoriteFood( 'curry' )}
-            value="curry"
-          />
-          <RadioButton
-            checked={favoriteFood === 'sushi'}
-            id="favoriteSushi"
-            label="Sushi"
-            name="favoriteFood"
-            onChange={() => setFavoriteFood( 'sushi' )}
-            value="sushi"
-          />
-        </Flex>
-      </Fieldset>
-    </Flex>
-  );
-}
-`}
+      <SlimBanner
```

Comment:
I moved all the docs over to RadioGroup to promote adoption and utilize the new docs